opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -1006,6 +1008,36 @@ def set_up_accelerate_config(
                 "Failed to open config file for ml common upload: " + file_path + "\n"
             )
 
+    def get_model_description_from_md_file(self, readme_file_path) -> str:
```

Comment:
Yes. As I mentioned above, I merged the other PR since it is needed for this workflow. I will migrate change to new branch to clean up the branch history before creating a final PR for this workflow.
<img width="1018" alt="Screenshot 2023-08-07 at 11 12 34 PM" src="https://github.com/opensearch-project/opensearch-py-ml/assets/106889996/12256cab-7ddf-4e50-b233-09882359c03e">